web/src/js/components/FlowView/Details.tsx

```diff
@@ -19,7 +26,19 @@ export function TimeStamp({t, deltaTo, title}) {
     )
 }
 
-export function ConnectionInfo({conn}) {
+type ConnectionInfoProps = {
```

Comment:
I think we should use https://github.com/mitmproxy/mitmproxy/blob/29997bca4b6436e5837c7d086d3311d6e79e9ca4/web/src/js/flow.ts#L35 here.

Reply:
The Connection interface didn't contain some fields used in ConnectionInfo component.
I added them to the Connection interface.